package-lock.json

```diff
@@ -1,6074 +1,113 @@
 {
   "name": "yt-channel-info",
   "version": "3.0.4",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Is this version downgrade intentional?

Reply:
Oops I missed that
Might be easier to just use yarn on all FT repos...

Reply:
@GilgusMaximus

Reply:
I removed the packagelock change